libsel4vm/src/arch/arm/vgic/vgic.c

```diff
@@ -551,7 +551,7 @@ static int vgic_dist_disable_irq(struct vgic_dist_device *d, vm_vcpu_t *vcpu, in
 {
     /* STATE g) */
     struct gic_dist_map *gic_dist = vgic_priv_get_dist(d);
-    if (irq >= 16) {
```

Comment:
Are you able to add a comment here that this is because 0-15 cannot be disabled.

Reply:
added a commit on top that adds a comment and also a debug log output